### ISSUE TITLE
Suggestions for graph visualization

### ISSUE THREAD

**Issue by nvaytet** [scipp/sciline#73]
After using sciline for a bit while developing the SANS direct beam, I've come accross a couple of things that are slightly annoying when trying to visualize the pipeline.

1. In Jupyter, since we moved to reprsenting the graph as an SVG where we can copy/paste the names in the boxes, it now always fits the graph to the width of the page, meaning that a full workflow graph will always be small and unreadable, while the old png repr was an large image you could navigate with a scrollbar. I think it would be nice to at least have the option of requesting a different output format for the graph visualization, e.g. `format='svg'` by default, with `'png'` available.

2. Another thing that would be really neat, before going to a full-blown interactive visualization of graphs, would be to make inspecting intermediate steps even easier. Instead of having to copy/paste the name of the quantity you want to inspect, then having to type `pipeline.compute(MaskedDirectBeam)`, if we could somehow make the boxes clickable which would then copy into the clipboard that `pipeline.compute(...)` code, so that you only need to paste in a cell?

I suspect this copy on click cannot be done without some javascript, which we might not want to have. Maybe an alternative would be to make them into links, where the link url wouldn't be a url, but `pipeline.compute(MaskedDirectBeam)` instead, and one could right click and do 'copy link address'?

**Comment by jl-wynen** [comment]
About point 2:

You would need to get the name of the pipeline from somewhere. We could of course simply use `pipeline` but that won't always work. Or we inspect the source code of the call to `visualize` and extract it from there. But that is can easily go wrong.

**Comment by SimonHeybrock** [comment]
I agree, I don't really see 2.) happening, too little utility for too much work and things that can go wrong.
